app/src/main/java/org/schabi/newpipe/fragments/list/comments/CommentRepliesFragment.java

```diff
@@ -110,7 +111,7 @@ protected Supplier<View> getListHeaderSupplier() {
             TextLinkifier.fromDescription(binding.commentContent, item.getCommentText(),
                     HtmlCompat.FROM_HTML_MODE_LEGACY, getServiceById(item.getServiceId()),
                     item.getUrl(), disposables, null);
-
+            binding.commentContent.setMovementMethod(LinkMovementMethod.getInstance());
```

Comment:
```suggestion
            binding.commentContent.setMovementMethod(LinkMovementMethodCompat.getInstance());
```

[`LinkMovementMethodCompat`](https://developer.android.com/reference/kotlin/androidx/core/text/method/LinkMovementMethodCompat) fixes an issue with links on Android versions below V.

Reply:
I think our [`LongPressLinkMovementMethod`](https://github.com/TeamNewPipe/NewPipe/blob/276bf390b2c669dfa02f8e907ab52b4425af84b4/app/src/main/java/org/schabi/newpipe/util/text/LongPressLinkMovementMethod.java) class should be used instead.

> [`LinkMovementMethodCompat`](https://developer.android.com/reference/kotlin/androidx/core/text/method/LinkMovementMethodCompat) fixes an issue with links on Android versions below V.

This change isn't the only place where `LinkMovementMethod` is used, you should probably open an issue for this :)

Reply:
I’ve updated the code to use LinkMovementMethodCompat for now, following the suggestion.
I’ll open a new issue soon to address replacing it with LongPressLinkMovementMethod.
Thanks a lot for the helpful suggestions!